src/compressed_tensors/quantization/lifecycle/initialize.py

```diff
@@ -234,6 +234,12 @@ def initialize_qparams(
         num_cols = strategy_cdiv(observed_shape[-1], block_structure[-1], strategy)
         expected_shape = (num_rows, num_cols)
 
+    elif strategy == QuantizationStrategy.ATTN_HEAD:
+        if len(observed_shape) < 2:
```

Comment:
a comment to indicate what the expected dimensions of `observed_shape` would be useful here, similar to what you've set [here](https://github.com/neuralmagic/compressed-tensors/pull/481/files#diff-14ea90f0da6e2343a0dee8b350b2d728f780d4a18c277bb65ce5457cfdc4da54R213)

something like this?
```suggestion
        # (..., head_dim, num_heads)
        if len(observed_shape) < 2:
```